.github/CODEOWNERS

```
@@ -3,15 +3,146 @@
+
+## package /packages/ens-controller
+/packages/ens-controller                    @MetaMask/confirmations
+/packages/ens-controller/package.json       @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/ens-controller/changelog.md       @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+
+## package /packages/eth-json-rpc-provider
+/packages/eth-json-rpc-provider        @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
+
+## package /packages/gas-fee-controller
+/packages/gas-fee-controller                @MetaMask/confirmations
+/packages/gas-fee-controller/package.json   @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/gas-fee-controller/changelog.md   @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+
+## package /packages/json-rpc-engine
+/packages/json-rpc-engine              @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
+
+## package /packages/json-rpc-middleware-stream
+/packages/json-rpc-middleware-stream   @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
+
+## package /packages/keyring-controller
+/packages/keyring-controller                     @MetaMask/accounts-engineers
+/packages/keyring-controller/package.json        @MetaMask/accounts-engineers @MetaMask/wallet-framework-engineers
+/packages/keyring-controller/changelog.md        @MetaMask/accounts-engineers @MetaMask/wallet-framework-engineers
+
+## package /packages/logging-controller
+/packages/logging-controller                @MetaMask/confirmations
+/packages/logging-controller/package.json   @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/logging-controller/changelog.md   @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+
+## package /packages/message-manager
+/packages/message-manager                   @MetaMask/confirmations
+/packages/message-manager/package.json      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/message-manager/changelog.md      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+
+## package /packages/name-controller
+/packages/name-controller                   @MetaMask/confirmations
+/packages/name-controller/package.json      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/name-controller/changelog.md      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+
+## package /packages/network-controller
+/packages/network-controller           @MetaMask/wallet-framework-engineers @MetaMask/metamask-assets
+
+## package /packages/notification-controller
+/packages/notification-controller              @MetaMask/notifications
+/packages/notification-controller/package.json @MetaMask/notifications @MetaMask/wallet-framework-engineers
+/packages/notification-controller/changelog.md @MetaMask/notifications @MetaMask/wallet-framework-engineers
```

Ideally our team would own all notifications, but I believe this is currently owned by the @MetaMask/snaps-devs

---

We have plans in the future to have a consolidation or unification of notifications (orchestration, creation, etc) :)

---

I don't want this as a blocker, so can leave it for now.

---

It does feel a bit premature to make this change until we have consolidated notifications more?

---

Agreed. Could we restore snaps-devs as co-codeowner for the time being?

```suggestion
/packages/notification-controller              @MetaMask/notifications @MetaMask/snaps-devs 
/packages/notification-controller/package.json @MetaMask/notifications @MetaMask/snaps-devs @MetaMask/wallet-framework-engineers
/packages/notification-controller/changelog.md @MetaMask/notifications @MetaMask/snaps-devs @MetaMask/wallet-framework-engineers
```